src/lambda/src/notifications/slack-manager.ts

```diff
@@ -0,0 +1,35 @@
+import { WebClient } from '@slack/web-api';
+import AWS from 'aws-sdk';
+
+import { NotificationPayload } from '../models';
+
+export class SlackManager {
```

Comment:
Extract an interface INotificationDeliveryClient
Rename this to SlackNotificationDeliveryClient
It should Have two method: 
SupportedChannel(ChannelType)
Send ( payLoad )

Reply:
Added interface and reworked manager class